cristin-biobank/src/main/java/no/unit/nva/biobank/model/nva/AssociatedProject.java

```diff
@@ -0,0 +1,75 @@
+package no.unit.nva.biobank.model.nva;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import no.unit.nva.biobank.model.cristin.CristinAssociatedProject;
+import no.unit.nva.commons.json.JsonSerializable;
+import nva.commons.core.JacocoGenerated;
+
+import java.beans.ConstructorProperties;
+import java.net.URI;
+import java.util.Map;
+import java.util.Objects;
+
+import static no.unit.nva.cristin.common.Utils.nonEmptyOrDefault;
+import static no.unit.nva.utils.UriUtils.createNvaProjectId;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AssociatedProject implements JsonSerializable {
+
+    private final String cristinProjectId;
+    private final Map<String,String> title;
+    private final URI url;
```

Comment:
id?

Reply:
changed to id and removed cristinProjectId as it is surplus